Dockerfile

```diff
@@ -62,6 +62,7 @@ ENV PATH="/opt/venv/bin:$PATH"
 
 COPY --chown=sync-engine:sync-engine ./ ./
 RUN \
+  --mount=type=cache,target=/home/sync-engine/.cache,uid=5000,gid=5000  \
```

Comment:
I'm guessing this one doesn't belong here?

Reply:
Yeah, haha. I use this to speed up repetitve installs locally but I didn't mean to commit it haha.